### PR TITLE
Metrics changes

### PR DESCRIPTION
**Pull request by eugskim**
*Issue #, if available:*

*Description of changes:*
Adding new metrics and tests

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by NikhilRaverkar** [comment]
Build is failing with an error: `No such file or directory: '/miniconda3/lib/python3.7/site-packages/numpy-1.21.6.dist-info/METADATA'`

Please [replace these lines](https://github.com/aws/sagemaker-xgboost-container/blob/master/docker/1.5-1/final/Dockerfile.cpu#L18-L21) with following:

```
RUN conda install numpy --force-reinstall && \
   rm -rf /miniconda3/lib/python3.7/site-packages/numpy-1.21.2.dist-info && \
    python3 -m pip install --no-cache /sagemaker_xgboost_container-1.0-py2.py3-none-any.whl && \
    python3 -m pip uninstall -y typing && \
    rm /sagemaker_xgboost_container-1.0-py2.py3-none-any.whl
```

File: docker/1.5-1/final/Dockerfile.cpu [hunk 1]
```diff
@@ -1,4 +1,4 @@
-ARG SAGEMAKER_XGBOOST_VERSION=1.5-1
+iARG SAGEMAKER_XGBOOST_VERSION=1.5-1
```
Comment (NikhilRaverkar) [inline docker/1.5-1/final/Dockerfile.cpu:1]: nit should be `ARG`

Comment (eugskim) [reply docker/1.5-1/final/Dockerfile.cpu:1]: Oh oops, sorry about that!

File: docker/1.5-1/final/Dockerfile.cpu [hunk 2]
```diff
@@ -15,7 +15,9 @@ RUN python3 -m pip install -r /requirements.txt && rm /requirements.txt
 # Copy wheel to container #
 ###########################
 COPY dist/sagemaker_xgboost_container-2.0-py2.py3-none-any.whl /sagemaker_xgboost_container-1.0-py2.py3-none-any.whl
-RUN rm -rf /miniconda3/lib/python3.7/site-packages/numpy-1.21.2.dist-info && \
+
+RUN conda install numpy --force-reinstall && \
```
Comment (yusharon) [inline docker/1.5-1/final/Dockerfile.cpu:19]: Why do we `force-reinstall` the numpy here?  Will it install the latest numpy version? Shouldn't we pin https://github.com/aws/sagemaker-xgboost-container/blob/master/requirements.txt#L11 here and use pinned numpy?

**Comment by eugskim** [comment]
Integration Tests for the new metrics (MAE and RMSE): [mxnet-algorithms-integ-test-220601-191722-724128]

File: docker/1.5-1/final/Dockerfile.cpu (hunk 2 above)
Comment (eugskim) [reply docker/1.5-1/final/Dockerfile.cpu:19]: Yes that is a good point. I'm not sure why the dockerfile got pushed. I'll remove it from the PR

**Comment by NikhilRaverkar** [comment]
Can you please confirm if the integ tests were run for an image containing these changes?

**Comment by eugskim** [comment]
> Can you please confirm if the integ tests were run for an image containing these changes?

Yes, the integration testing was ran on a docker image with the new changes in this PR

**Review by NikhilRaverkar — APPROVED** [review]

File: docker/1.5-1/final/Dockerfile.cpu (hunk 2 above)
Comment (eugskim) [reply docker/1.5-1/final/Dockerfile.cpu]: The change has been added!

**Review by dewan-c — APPROVED** [review]

**Pull request merged**